Makefile

```diff
@@ -15,6 +15,36 @@ ut: update-version
 	calico/test \
 	nosetests tests/unit -c nose.cfg
 
+# Run system tests.
+st: docker-image run-etcd run-k8s-apiserver
```

Comment:
Yes in principle, but I don't see how I can neatly do that until #86 has been merged first.  I'd favour leaving that for a follow-on change.